src/DeleteComment.jsx

```diff
@@ -0,0 +1,26 @@
+import { useContext, useEffect, useState } from "react";
+import { deleteComment } from "./apis";
+import { UserContext } from "./User";
+
+export const DeleteComment = ({author, comment_id, setListOfComments, listOfComments, setIsLoading}) => {
+    const { user, isLoggedIn } = useContext(UserContext)
+
+    const deleteCommentById = () => {
+        const commentListDeleted = listOfComments.filter((comment) => {return comment.comment_id !== comment_id})
+        const commentList = [...listOfComments]
+        setIsLoading(true)
+        setListOfComments(commentListDeleted)
+        deleteComment(comment_id).then(() => {
+            setIsLoading(false)
+            alert("Comment Deleted")
```

Comment:
Don't worry about changing this unless you have time to go back as the alert method works but it might be better to use a p tag here to display the confirmation/error to the user just becuase alerts are synchronous and stop the user doing anything until they close the message.